recipes/libbacktrace/all/patches/cci.20240719-0001-pointer-arithmetic.patch

```diff
@@ -0,0 +1,31 @@
+diff --git a/pecoff.c b/pecoff.c
+index 636e1b1..975e417 100644
+--- a/pecoff.c
++++ b/pecoff.c
+@@ -704,7 +704,7 @@ coff_add (struct backtrace_state *state, int descriptor,
+       magic_ok = memcmp (magic, "PE\0", 4) == 0;
+       fhdr_off += 4;
+ 
+-      memcpy (&fhdr, fhdr_view.data + 4, sizeof fhdr);
++      memcpy (&fhdr, (char*)fhdr_view.data + 4, sizeof fhdr);
```

Comment:
Is this needed to compile without warnings?
Is there any open PR on upstream to link it with this patch?

Thank you in advance @toge 💯

Reply:
@perseoGI 
Sorry. I can't find any information about it.
I just follow cci.20210118 patch.

Reply:
Okey, then I'll approve. 
Nevertheless, it will be great to notify upstream about than warning!